From fdb3efa5ddfa7120de98f8ae78b5f5dc9e8e2e71 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Tue, 18 Feb 2014 11:04:15 -0500
Subject: [PATCH] add flock64 alias for (struct) flock in fcntl.h

this was a missing part of the LFS64 API; it's "needed" for use with
fcntl and the corresponding lock commands.
---
 include/fcntl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/fcntl.h b/include/fcntl.h
index e01a4d1..2d8fa6e 100644
--- a/include/fcntl.h
+++ b/include/fcntl.h
@@ -161,6 +161,7 @@ ssize_t tee(int, int, size_t, unsigned);
 #define F_GETLK64 F_GETLK
 #define F_SETLK64 F_SETLK
 #define F_SETLKW64 F_SETLKW
+#define flock64 flock
 #define open64 open
 #define openat64 openat
 #define creat64 creat
-- 
1.8.5.4

